js/scripts.js

```diff
@@ -1,5 +1,6 @@
 // scripts.js
 
-var name = prompt('Enter your name');
-alert('Hello, ' + name);
-console.log('Hello, ' + name);
\ No newline at end of file
+var name = prompt('Enter your name');//This is my first js comment
+/*alert('Hello, ' + name);*/
+console.log('Hello, ' + name);//was this exercise really just about 
+//inserting comments?
```

Comment:
yes it was :)
zebym byl pewny ze wszystko wiesz: dodaj komentarz ktory bedzie komentowal wiecej niz jedna linijke naraz